spec/file_reference_installer/copy_xcassets_file_referenc_installer_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.